src/main/java/com/schematica/client/util/RotationHelper.java

```diff
@@ -27,6 +27,10 @@
 
 public final class RotationHelper
 {
+    /**
+     * This is needed for Schematica to work.
+     */
+    @SuppressWarnings("deprecation")
     private static final FMLControlledNamespacedRegistry<Block> BLOCK_REGISTRY = GameData.getBlockRegistry();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getBlockRegistry"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)